README.md

```diff
@@ -39,20 +39,34 @@ It will serve the following Composer repository at `/packages.json` automagicall
 
 ## Installation
 
-Create the project:
+### 1. Create the project:
 
 ```
 composer create-project rarst/release-belt
 ```
 
-Place release ZIPs into `/releases/[type]/[vendor]/`.
+### 2. Place release ZIPs into `/releases/[type]/[vendor]/`. 
+`[type]` could be e.g. "symfony-bundle", "wordpress-plugin or "typo3-cms-extension".
 
-Configure a web server to serve `index.php`, for example with the following `.htaccess`:
+### 3. Configure a web server to serve `index.php`
+
+For example with the following `.htaccess`:
 
 ```
 FallbackResource /index.php
 ```
 
+If your server does not support FallbackResource, you can use mod_rewrite in your `.htaccess` with this code:
+
+```
+RewriteEngine On
+RewriteCond %{REQUEST_FILENAME} !-f
+RewriteCond %{REQUEST_FILENAME} !-d
+RewriteRule . /index.php [L]
+```
+Make sure, that the folder, which contains the `index.php` has FTP-Permissions `755`
```

Comment:
Permissions are way too specific to hosting, what fixes it on one host breaks it on another. Please drop this line.

Reply:
The problem I had is, that packages.json gave me a 404-error, even though I did the installation correctly. 

Could we then add a troubleshooting-section, or can we add it to the FAQ? "What do I do, if packages.json shows me a 404 error?"

Reply:
I added a troubleshooting section. What do you think?

Reply:
In my opinion it's not up to _this_ project to teach anyone to configure a web server really. There are a lot of ways to run PHP out there, and as above they are full of nuances from mutually incompatible to creating security holes.

I would still prefer you just drop mention of folder permissions.

For the troubleshooting purposes I would just say "If you have further trouble with configuration please refer to your hosting documentation or support for recommended settings".

Reply:
I removed the troubleshooting section. I think it confuses more than it helps.